TTPChallenge/DataAnalystVC.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  DataAnalystVC.swift
+//  TTPChallenge
+//
+//  Created by Mirim An on 2/15/17.
+//  Copyright © 2017 TeamMDC. All rights reserved.
+//
+
+import UIKit
+
+class DataAnalystVC: UIViewController {
+
+    @IBOutlet var informationButtons:[UIButton]!
+    
+    @IBOutlet weak var retakeQuizButton: UIButton!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        for button in informationButtons {
+            button.layer.cornerRadius = 12
```

Comment:
make this into a User Defined Runtime Attribute on your storyboard.

You do this by clicking onto "show Identity Inspector" on the right hand toolbar (that newspaper looking thing). Your "Key Path" would be layer.cornerRadius, your type would be "Number" and your value would be 12